app/src/test/java/com/example/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -13,46 +13,32 @@ import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.rules.TestRule
 
-
-
```

Comment:
According to Kotlin best practices, there should only be a single blank line seperating these elements (Not a code review suggestion).